tasks/global-setup.yml

```diff
@@ -55,6 +55,20 @@
     - restart_gitlab_runner
     - restart_gitlab_runner_macos
 
+- name: Add log_level to config
+  lineinfile:
+    dest: "{{ gitlab_runner_config_file }}"
+    regexp: ^log_level =
+    line: log_level = "{{ gitlab_runner_log_level }}"
+    insertbefore: BOF
+    state: present
+  no_log: "{{ gitlab_runner_no_log_secrets | default(omit) }}"
+  when: gitlab_runner_log_level is defined # Ensure value is set
```

Comment:
```
# Ensure value is set
```

is not needed. People working with Ansible know what `is defined` menas

Reply:
You can also remove the comments from the other lines if you want to.

Reply:
Done! I've removed the comment on line 66, as well as all other instances of this comment in this file.

Reply:
Done! I've removed the comment on line 66, as well as all other instances of this comment in this file.